### PR TITLE
dvisvgm: update to 2.6.1

### PR DESCRIPTION
**Pull request by mojca**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.13.6 17G3025
Xcode 10.1 10B61 

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

### CODE REVIEW

File: graphics/dvisvgm/Portfile [hunk 1]
```diff
@@ -13,24 +14,23 @@ description         DVI to SVG converter
...
 
-depends_lib         port:freetype \
+depends_lib         port:brotli \
+                    port:freetype \
                     port:ghostscript \
                     port:potrace \
                     port:texlive-bin \
+                    port:woff2 \
                     port:zlib
+                    # port:xxhash
```
Comment (pmetzger) [inline graphics/dvisvgm/Portfile]: Not sure what this bit is for? Perhaps you should comment the comment?

**Comment by pmetzger** [comment]
LGTM.

File: graphics/dvisvgm/Portfile (hunk 1 above)
Comment (mojca) [reply graphics/dvisvgm/Portfile]: This is a dependency which we are not packaging yet, so it uses the library shipped with the sources.

Comment (pmetzger) [reply graphics/dvisvgm/Portfile]: Right, but that could have been mentioned in a comment in the portfile.

Comment (mojca) [reply graphics/dvisvgm/Portfile]: What's wrong with the (non-outdated version of) the current comment?

Comment (pmetzger) [reply graphics/dvisvgm/Portfile]: I got confused by the github interface and didn't see that it had changed. Apologies.

**Pull request merged**